dev/com.ibm.ws.java.level_fat/src-reference/java17/org/gjwatts/liberty/TestApp.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.gjwatts.liberty;
```

Comment:
I would maybe re-iterate here as a note that changes to the application code will not be honored, and that the application is build externally and uploaded as a binary.

Reply:
Thanks for the suggestion.  I have added this to the FAT README.txt file.